test/dev/integration.js

```diff
@@ -465,3 +466,29 @@ test('[now dev] temporary directory listing', async t => {
     dev.kill('SIGTERM');
   }
 });
+
+test.only('[now dev] no build matches warning', async t => {
+  const directory = fixture('no-build-matches');
+  const { dev, port } = testFixture(directory, {
+    stdio: ['ignore', 'pipe', 'pipe']
+  });
+
+  try {
+    // start `now dev` detached in child_process
+    dev.unref();
+
+    dev.stderr.setEncoding('utf8');
+
+    await new Promise(resolve => {
+      dev.stderr.on('data', str => {
+        if (str.includes('did not match any source files')) {
```

Comment:
Could it be possible that `str` is cut somewhere in between, because the chunk is too small?

cc @sophearak this is how we could await `READY`

Reply:
In theory it's possible, but in practice it doesn't happen since in now-cli the message is written using a single `process.stderr.write()` invocation.

If this test starts to become flaky then we can add line buffering to the test case.